op-batcher/batcher/channel_manager.go

```diff
@@ -260,6 +260,10 @@ func (s *channelManager) processBlocks() error {
 		if s.currentChannel.IsFull() {
 			break
 		}
+		if s.rollupCfg.KromaMPTTime != nil && block.Time() == *s.rollupCfg.KromaMPTTime-s.rollupCfg.BlockTime {
+			s.currentChannel.channelBuilder.setFullErr(errors.New("reached the block just before KromaMPTTime"))
```

Comment:
Since setFullErr is a managed function of the channelBuilder, why not handle this logic in the AddBlock of the channelBuilder?